frontend/admin/src/js/components/user/GeneralInformationTab/GeneralInformationTab.tsx

```diff
@@ -271,6 +272,37 @@ const AboutSection: React.FC<BasicSectionProps> = ({ user }) => {
         })}
       </span>
       <div data-h2-flex-item="b(2of3) l(1of3)">{user.telephone}</div>
+
+      <span data-h2-flex-item="b(1of3) l(1of6)">
+        {intl.formatMessage({
+          defaultMessage: "Member of CAF:",
+          description: "label for CAF status",
+        })}
+      </span>
+      <div data-h2-flex-item="b(2of3) l(1of3)">
+        {user.isVeteran === true &&
+          intl.formatMessage({
+            defaultMessage: "Is a veteran or member",
+            description: "user is a veteran or member",
+          })}
+        {user.isVeteran === false &&
+          intl.formatMessage({
+            defaultMessage: "Is not a veteran or member",
+            description: "user is not a veteran or member",
+          })}
+      </div>
```

Comment:
Do we have a null state for this? Looks a little odd when it is not set:

<img width="895" alt="Screen Shot 2022-08-12 at 3 16 16 PM" src="https://user-images.githubusercontent.com/4127998/184428372-d0996588-d1b1-4730-b5fa-1372a2676880.png">

Reply:
For that one, I was mimicking the others, they show to admin that the user didn't complete it. Which looks like an intentional design decision. 
Let me know if you still feel iffy on it and I will ask someone

![image](https://user-images.githubusercontent.com/40485260/184431471-92bb0b4c-a844-4e08-82ae-af6a10032728.png)

Reply:
they all look like that actually

![image](https://user-images.githubusercontent.com/40485260/184432058-8945c6fd-8a6b-44c0-9e9e-7dab98922413.png)

Reply:
Okay, if that is the case then I think we should be fine 👍